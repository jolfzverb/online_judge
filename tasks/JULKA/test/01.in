10
2
10
10
0
0
114
6
2003
65
1000
998
5555
5555
10
2
10
2
10
2
