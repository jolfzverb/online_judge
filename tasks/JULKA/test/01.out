6
4
10
0
0
0
60
54
1034
969
999
1
5555
0
6
4
6
4
6
4
